crates/workspace/src/workspace.rs

```diff
@@ -2510,10 +2513,29 @@ impl Workspace {
         self.zoomed_position = None;
         cx.emit(Event::ZoomChanged);
         self.update_active_view_for_followers(cx);
+        self.track_alternate_files(cx);
 
         cx.notify();
     }
 
+    fn track_alternate_files(&mut self, cx: &mut ViewContext<Self>) {
+        if let Some(item) = self.active_item(cx) {
+            if let Some(project_path) = item.project_path(cx) {
+                let (current, alternative) = self.alternate_file_paths.clone();
```

Comment:
You can avoid cloning `alternative_file_paths` by borrowing it instead:

```rs
 fn track_alternate_files(&mut self, cx: &mut ViewContext<Self>) {
    if let Some(item) = self.active_item(cx) {
        if let Some(project_path) = item.project_path(cx) {
            let (current, alternative) = &self.alternate_file_paths;
            match current {
                Some(current) => {
                    if *current != project_path {
                        self.alternate_file_paths = (Some(project_path), Some(current.clone()));
                    }
                }
                None => {
                    self.alternate_file_paths = (Some(project_path), None);
                }
            }
        }
    }
}
```

Reply:
Oh, yeah that's better! Thanks, I fixed it now.